sklearn/utils/tests/test_arpack.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+from sklearn.utils._arpack import _init_arpack_v0
+
+
+def test_init_arpack_v0():
+    v0s = []
+    for i in range(100):
+        v0s.append(_init_arpack_v0(1000, i))
+        if i > 0:
+            assert not any(np.equal(v0s[i], v0s[i-1]))
```

Comment:
I do not think we need to test that all the adjacent arrays are not equal.


 We could add a test to make sure the values are between -1 and 1.

Reply:
I have added check for checking range between -1 and 1.

Reply:
do you want me to remove adjacent array check altogether? or may be we should just check for 2 adjacent?

Reply:
I do not think we need to test:

```py
assert not any(np.equal(v0s[i], v0s[i-1]))
```

This is basically testing that `rng.uniform` generates different values.

Reply:
ok, removed